scripts/license_check.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+FILES=$(find . -type f -name "*.go")
```

Comment:
We should skip `testdata`, `third-party` dirs ala [helm](https://github.com/helm/helm/blob/master/scripts/validate-license.sh).

Reply:
Sounds good...